contrib/readers/empty-jquery-reader.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: reader
+metadata:
+  id: test/empty-jquery-reader
+  version: 0.0.1
+  name: Empty JQuery Reader
+  description: "Empty JQuery reader for testing purposes; Page Editor used to product reader definition"
```

nit: `product` -> `produce`
```suggestion
  description: "Empty JQuery reader for testing purposes; Page Editor used to produce reader definition"
```